base/src/main/scala/co/uproot/abandon/Ast.scala

```diff
@@ -272,8 +272,10 @@ sealed class ASTTangibleEntry extends ASTEntry
 
 case class Transaction(pos: InputPosition, date: Date, posts: Seq[Post], annotationOpt: Option[String], payeeOpt: Option[String], comments: List[String]) extends ASTTangibleEntry
 
-case class Definition(pos: InputPosition, name: String, params: List[String], rhs: Expr) extends ASTTangibleEntry {
-  def prettyPrint = "def %s(%s) = %s" format (name, params.mkString(", "), rhs.prettyPrint)
+case class Definition(pos: InputPosition, name: String, params: List[String], rhs: Expr, private var used: Boolean = false) extends ASTTangibleEntry {
```

Comment:
Could the `used` constructor parameter be changed to a member variable? If it's part of the constructor parameter list, it becomes a part of its identity.